docs/en/guide/mount_at_boot.md

```diff
@@ -27,6 +27,42 @@ sudo chkconfig --add netfs
 
 :::
 
+### Automating Mounting with systemd.mount
```

Comment:
Please add the Chineses version as well

Reply:
Sure, missing Chinese version added now.